uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-popup/details-tab/common.html

```diff
@@ -25,29 +25,18 @@
   <div class="col-xs-12 form-horizontal">
     <div class="form-group">
       <label class="col-sm-4 control-label required-mark">From Date</label>
-      <div class="col-xs-6 col-sm-4 chr_leave-request-date-picker-container">
-        <div class="input-group">
-          <input
-            type="text"
-            class="form-control"
-            uib-datepicker-popup
-            show-button-bar="false"
-            datepicker-options="detailsTab.uiOptions.date.from.options"
-            ng-model="detailsTab.uiOptions.fromDate"
-            is-open="detailsTab.uiOptions.date.from.show"
-            ng-readonly="true"
-            ng-disabled="!detailsTab.isRole('admin') && (detailsTab.isMode('view') || (detailsTab.isRole('manager') && !detailsTab.isMode('create') && !detailsTab.isLeaveType('sickness')))"
-            ng-click="detailsTab.uiOptions.date.from.show = true"
-            ng-change="detailsTab.updateAbsencePeriodDatesTypes(detailsTab.uiOptions.fromDate, 'from')"/>
-          <span
-            class="input-group-addon pointer"
-            ng-disabled="!detailsTab.isRole('admin') && (detailsTab.isMode('view') || (detailsTab.isRole('manager') && !detailsTab.isMode('create') && !detailsTab.isLeaveType('sickness')))"
-            ng-click="detailsTab.uiOptions.date.from.show = true">
-            <i class="fa fa-calendar"></i>
-          </span>
-        </div>
+      <div class="col-sm-4 chr_leave-request-date-picker-container">
+        <date-picker
+          datepicker-options="detailsTab.uiOptions.date.from.options"
+          show-button-bar="false"
+          ng-model="detailsTab.uiOptions.fromDate"
+          is-open="detailsTab.uiOptions.date.from.show"
+          ng-readonly="true"
+          ng-disabled="!detailsTab.isRole('admin') && (detailsTab.isMode('view') || (detailsTab.isRole('manager') && !detailsTab.isMode('create') && !detailsTab.isLeaveType('sickness')))"
+          ng-change="detailsTab.updateAbsencePeriodDatesTypes(detailsTab.uiOptions.fromDate, 'from')">
```

Comment:
is this line necessary?
`ng-click="detailsTab.uiOptions.date.from.show = true"` ?

Reply:
I dont see this line, maybe you checked on a previous commit?